sdk/python/pvsite_datamodel/write/generation.py

```diff
@@ -34,6 +34,19 @@ def _upsert(session: Session, table: Base, rows: list[dict]):
     session.execute(stmt, rows)
 
 
+def _upsert_do_nothing(session: Session, table: Base, rows: list[dict]):
```

Comment:
Doesn't really matter, but this should probably be called "insert" not "upsert", since it will never do the "update" part of an "upsert"

Reply:
yep, ill change that

Reply:
Maybe I'd change the name to `_insert_if_not_present` or `_insert_if_not_exists`